ocs_ci/utility/kms.py

```diff
@@ -578,7 +593,7 @@ def vault_create_policy(self, policy_name=None):
             raise VaultOperationError(
                 f"Failed to create policy f{self.vault_policy_name}"
             )
-        self.vault_path_token = self.generate_vault_token()
+        self.vault_path_token = self.set_vault_token(self)
```

Comment:
If you go the classmethod route (which we should), you wont need to pass self here.
```suggestion
        self.vault_path_token = self.set_vault_token()
```